core/testing/runner.odin

```diff
@@ -61,6 +63,18 @@ get_log_level :: #force_inline proc() -> runtime.Logger_Level {
 	}
 }
 
+JSON :: struct {
+	total:    int,
```

Comment:
I would have given this a more descriptive name, like `JSON_Report` and changed the config to `EXPORTING_JSON`.